src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/DescriptionAddendum.kt

```diff
@@ -0,0 +1,69 @@
+package uk.gov.justice.digital.hmpps.incidentreporting.jpa
+
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.ManyToOne
+import org.hibernate.Hibernate
+import uk.gov.justice.digital.hmpps.incidentreporting.jpa.helper.EntityOpen
+import java.time.LocalDateTime
+import uk.gov.justice.digital.hmpps.incidentreporting.dto.DescriptionAddendum as DescriptionAddendumDto
+
+@Entity
+@EntityOpen
+class DescriptionAddendum(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long? = null,
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  val report: Report,
+
+  val createdBy: String,
```

Comment:
Add firstName and lastName

Reply:
done here: https://github.com/ministryofjustice/hmpps-incident-reporting-api/pull/313/commits/3047e4c3a02d81617abe4eda44953f78c04c687a